README.md

```diff
@@ -7,7 +7,16 @@ In order to setup concourse initially on a cluster, you will need to have `terra
 
 1. Edit `resources/secrets.tf` and add a configuration block for the new cluster, if one does not already exist.
 
-2. `./bin/setup-on-cluster <cluster-name>` where `cluster-name` is the name of the cluster you want to install Concourse CI on, as it appears in the terraform workspaces [here](https://github.com/ministryofjustice/kubernetes-investigations/tree/master/terraform/cloud-platform).
+## Prerequisite before running step 2.
```

Comment:
I'm re-reading this again. I think this entire section could do with some re-writing (feels more complicated than it should be). Essentially:
1. `kubectl config use-context`
2. `terraform workspace select`
3. edit `secrets.tf` if needed
4. run terraform

(as a side note, I think we should stop repeating the `terraform init / plan/ apply` song in every other repo :) )

Reply:
Agree, will update the README, Thanks

Reply:
> I'm re-reading this again. I think this entire section could do with some re-writing (feels more complicated than it should be). Essentially:
> 
> 1. `kubectl config use-context`
> 2. `terraform workspace select`
> 3. edit `secrets.tf` if needed
> 4. run terraform
> 
> (as a side note, I think we should stop repeating the `terraform init / plan/ apply` song in every other repo :) )

Update the README as suggested